examples/tf/te_ppo_metaworld_mt10.py

```diff
@@ -48,7 +48,6 @@ def te_ppo_mt10(ctxt, seed, n_epochs, batch_size_per_task):
     policy_ent_coeff = 2e-2
     encoder_ent_coeff = 2e-4
     inference_ce_coeff = 5e-2
-    max_episode_length = 100
```

Comment:
add this to GymEnv above

Reply:
So for all the RL2 and metaworld examples we have to manually specify a `max_episode_length`? ie.
`GymEnv(MT10.from_task(task), max_episode_length=100)`

Reply:
I don't think metaworld has max_episode_length populated, so it probably defaults to infinity.

Reply:
Similarly for HalfCheetah

Reply:
Ah got it.